content/guides/quickstart/chat-guide.md

```diff
@@ -1272,6 +1565,106 @@ handleUpdate = upd => {
+```
+
+We also need to specify which agents to start when our desk is installed. We do
+this in a `desk.bill` file:
+
+```shell {% copy=true %}
+echo "~[%hut]" > desk.bill
+```
+
+Lastly, we need to create a Docket file. Docket is the agent that manages app
+front-ends - it fetches & serves them, and it also configures the app tile and
+other metadata. Create a `desk.docket-0` file in the `hut` directory and add the
+following:
+
+```hoon {% copy=true %}
+:~
+  title+'Hut'
+  info+'A simple chat app.'
+  color+0x7c.afc2
+  version+[0 1 0]
+  website+'https://urbit.org'
+  license+'MIT'
+  base+'hut'
+  glob-ames+[~sampel-sampel-sampel-sampel--sampel-sampel-sampel-samzod 0v0]
+==
+```
+
+The main field of note is `glob-ames`. A glob is the bundle of front-end
+resources (our React app), and the `-ames` part means it'll be distributed via
+the normal inter-ship networking protocol, as opposed to `glob-http` where it
+would be fetched from a separate server. The two fields are the ship to fetch it
+from and the hash of the glob. We can get the full name of our comet by typing
+`our` in the Dojo (don't use the ship name above, it's just a sample). We're
+going to upload the glob in the next step, so we'll leave the hash as `0v0` for
+the moment.
+
+## Put it together
+
+Our app is now complete, so let's try it out. In the Dojo of our comet,
+we'll create a new desk by forking from an existing one:
+
+``` {% copy=true %}
+|merge %hut our %webterm
+```
+
+Next, we'll mount the desk so we can access it from the host OS:
+
+``` {% copy=true %}
+|mount %hut
+```
+
+Currently its contents are the same as the `%webterm` desk, so we'll need to
+delete those files and copy in our own instead. In the normal shell, do the
+following:
+
+```shell {% copy=true %}
+rm -r dev-comet/hut/*
+cp -r hut/* dev-comet/hut/
+```
+
+Back in the Dojo again, we can now commit those files and install the app:
+
+``` {% copy=true %}
+|commit %hut
+|install our %hut
+```
+
+The last thing to do is upload our front-end resources. Open a browser and go
+to `localhost:8080` (or just `localhost` on a Mac). Login with the comet's web
+code, which you can get by running `+code` in the Dojo. Next, go to
+`localhost:8080/docket/upload` (or `localhost/docket/upload` on a Mac) and
+it'll bring up the Docket Globulator tool. Select the `hut` desk from the
+drop-down menu, then navigate to `hut-ui/build` and select the whole folder.
+Finally, hit `glob!` and it'll upload our React app.
+
+If we return to `localhost:8080` (or `localhost` on a Mac), we should see a
+tile for the Hut app. If we click on it, it'll open our React front-end and we
+can start using it.
+
+One thing we can also do is publish the app so others can install it from us.
+To do so, just run the following command:
+
+``` {% copy=true %}
+:treaty|publish %hut
+```
+
+Now our friends will be able to install it with `|install <our ship> %hut` or by
+searching for `<our ship>` on their ship's homescreen.
+
 ## Next steps
```

Comment:
Consider something like this:

"Although a minimalist demonstration app, a more full-featured implementation of a chat app could build on this by adding metadata about users (contacts), supporting more bells and whistle (like reactions and multimedia embedding), and additional security features."